integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -301,4 +304,45 @@ object CommonUtil {
       LOGGER.info(s"mapreduce.input.fileinputformat.split.maxsize: ${ newSplitSize.toString }")
     }
   }
+
+  def getCsvHeaderColumns(carbonLoadModel: CarbonLoadModel): Array[String] = {
+    val delimiter = if (StringUtils.isEmpty(carbonLoadModel.getCsvDelimiter)) {
```

Comment:
I think delimiter can not be " ", right? so better to use isBlank instead of isEmpty